pkg/controller/dataimportcron-controller.go

```diff
@@ -1254,6 +1254,7 @@ func (r *DataImportCronReconciler) newSourceDataVolume(cron *cdiv1.DataImportCro
 	dv.Namespace = cron.Namespace
 	r.setDataImportCronResourceLabels(cron, dv)
 	cc.AddAnnotation(dv, cc.AnnImmediateBinding, "true")
+	cc.AddAnnotation(dv, AnnLastUseTime, time.Now().UTC().Format(time.RFC3339Nano))
```

Comment:
At first I was worried older versions won't pass annotations but the problem was only with labels. https://github.com/kubevirt/containerized-data-importer/issues/2533